cmd/osbuild-composer/main.go

```diff
@@ -49,6 +58,14 @@ func main() {
 	}
 
 	switch config.LogFormat {
+	case "journal":
+		if journal.Enabled() {
+			logrus.SetFormatter(&logrus.JSONFormatter{})
+			logrus.AddHook(&common.JournalHook{})
+			logrus.SetOutput(io.Discard)
+		} else {
+			logrus.SetFormatter(&logrus.TextFormatter{})
+		}
```

Comment:
Fallback to "text" setting if systemd is not present.

Reply:
The comment would be more useful in the code instead of the in-line PR comments.